dashboard/modules/train/train_head.py

```diff
@@ -96,6 +99,19 @@ async def get_train_runs(self, req: Request) -> Response:
             content_type="application/json",
         )
 
+    def _update_actor_status(self, train_runs):
+        actor_status_table = {actor.actor_id: actor.state for actor in list_actors()}
```

Comment:
I found this `list_actors` api that can return an actor table, which makes the code cleaner. Not sure if there's any performance issue.

Reply:
the state-api list actors uses http to request state api from state_head.py.

I think we can have better performance if we use a GRPC call to get the actor data like the `_update_actors` call in `actor_head.py`

Reply:
Got it! Let me take a look and replace it with a grpc call.